src/components/ens-registration/RegistrationAvatar/RegistrationAvatar.tsx

```diff
@@ -186,7 +184,9 @@ const RegistrationAvatar = ({
                 ? onShowExplainSheet
                 : isTesting
                 ? handleSelectNFT
-                : undefined
+                : enableNFTs
+                ? undefined
+                : handleSelectImage
```

Comment:
I don't like that nested ternary, any chence of moving it somewhere else or making a wrapper handler that picks what to call with if else statements?